### DISCUSSION TITLE
Regarding benchmarks

### DISCUSSION THREAD

**Discussion by AbdulWMughal** [weiznich/diesel_async#167] [Q&A]
Looking at the diesel benchmark metrics i was shocked at how poor tokio_Postgres performed. Any ideas as to why? For medium complex queries, inserts, associations it seemed to do poorly with large load of queries. This has me baffled. Perhaps manually creating a transaction and using raw versions (query_raw) may be the solution. What do you think?

https://github.com/diesel-rs/metrics/

**Comment by weiznich** [comment]
I did not perform any in depth investigations on the performance of non-diesel crates in there, so I cannot comment on that.
